sherlock/__main__.py

```diff
@@ -22,10 +22,7 @@
 
     python_version = str(sys.version_info[0])+"."+str(sys.version_info[1])+"."+str(sys.version_info[2])
 
-    if major != 3:
-        print("Sherlock requires Python 3.6+\nYou are using Python %s, which is not supported by Sherlock" % (python_version))
-        sys.exit(1)
-    if minor < 6:
+    if major != 3 or major == 3 and minor < 6:
```

Comment:
Since you have `if major != 3`, doesn't that mean Sherlock will not work on python4? I know Python4 is not out yet, but I have a feeling it will come out very soon.

Reply:
Yeah, that makes sense. But, I think it is better to wait for Python4 to come out and test Sherlock with it, after which the condition can be changed to ```major < 3 or major == 3 and minor < 6```.